hack/logextractor.sh

```diff
@@ -2,20 +2,52 @@
 #
 # This script can be used to extract the failure cluster logs tarball from the PVC created for an install pod.
 #
```

Comment:
Can we add a sentence here to mention that we need a failed install or install in progress for this script work?

Reply:
Is that not implied by "extract the failure cluster logs tarball from the PVC created for an install pod"?

Reply:
Nope, at least to me. +1 for explicitly mentioning it